app/styles/components/bc-select.scss

```diff
@@ -251,7 +251,7 @@ $off-white: #f5f5f5;
 		&.small {
 			& > .list-container {
 				& > .select-container {
-					width: calc(100vw - 20px);
+					width: calc(100vw - 60px);
```

Comment:
This is not a solution to the issue. The issue is caused by the location on the screen so if the button is moved further to the left the issue would still exist. The dropdown is supposed to go full screen on mobile and the 20px is to match the mobile padding of 10px left and right.